src/components/NcButton/NcButton.vue

```diff
@@ -478,22 +635,30 @@ export default {
 		},
 	},
 
+	mounted() {
+		// TODO: Remove with stable release
+		if (this.$slots?.default && this.$slots.default[0]?.text && this.$slots.default?.isComment !== true) {
```

Comment:
This is guaranteed that `this.$slots` exists on a Vue component instance.
```suggestion
		// TODO: Remove with stable release
		if (this.$slots.default && this.$slots.default[0]?.text && this.$slots.default?.isComment !== true) {
```